earth_enterprise/src/common/serverdb/serverdbReader.cpp

```diff
@@ -29,6 +29,52 @@
 #include "common/geGdalUtils.h"
 #include "common/generic_utils.h"
 
+namespace {
+  std::string JsToJson(const std::string& in)
```

Comment:
Let's add a TODO comment stating where this code was copied [from](https://github.com/google/earthenterprise/blob/master/earth_enterprise/src/fusion/autoingest/JsUtils.cp) and refactoring is needed to remove the code duplication.